pagopa/io-app
# chore(Cross): [IOAPPX-277] Update check_urls script blacklist

## Short description
This PR updates the `check_urls` scrips blacklist.

## List of changes proposed in this pull request
- Remove old links which are not used anymore;
- Add new urls which currently cause false positive results.

## How to test
With `python` our your python virtual environment of choice run `pip install --user pipenv` and then `python check_urls`. No false positive should be included in the results list.
Note: sometimes the check if flaky, thus it might still return some false positive but it shouldn't return anything in the blacklist.
